### ISSUE TITLE
LaserScanReader.cs Bug, Improper radians to degrees conversion

### ISSUE THREAD

**Issue by E-Leask** [siemens/ros-sharp#361]
* [x] I am at the right place and my issue is directly related to ROS#. General technical questions I would post e.g. at [ROS Answers](https://answers.ros.org/) or [Stack Overflow](https://stackoverflow.com). For library-specific questions I would look for help in the corresponding library forums.
* [x] I have thoroughly read [the Contributing Guideline](Contributing.md) and writing this issue is the right thing to do in my case.
---
## I found a bug! ##
* [x] I am using the latest ROS# version available here on the master branch.
* [x] I am adding all required information, code and data files, screenshots and log files so that you can reproduce the problem.

* My OS is: Windows 10
* My Unity Version is: 2019.4.11.f1
* My ROS Distribution is: NA
* My Build Target Platform is: NA

**Here is my bug description:**
Issue related line 68 of LaserScanReader.cs 
`rays[i] = new Ray(transform.position, Quaternion.Euler(new Vector3(0, angle_min - angle_increment * i * 180 / Mathf.PI, 0)) * transform.forward);`

Specifically `angle_min - angle_increment * i * 180 / Mathf.PI` is not calculated properly. This is supposed to convert the angle from radians to degrees.
`angle_min` and  `angle_increment` inputs are in radians. 
 
Perform the following steps reproduce the bug:
Set values of LaserScanReader component to 
`angle_min=-3.14` 
`sample=360`
`angle_increment=0.01745329`
`angle_max=0.01745329`

Observed results:
Read the output `angle_min - angle_increment * i * 180 / Mathf.PI` calculation and you'll see instead that it starts from -3.14 degrees 
Another sanity check, using the values listed above. An object directly infront of the laserScanReader you would expect values in the middle of the `Ranges[]` output. While currently you would find them are the beginning/end of the `Range[]` array.

Expected results:
We should expect laserScanReader to start from -3.14 (-180) and iterate through to 3.14(180)

Relevant Code:
This would solve the problem(you probably could do something a little nicer with the brackets)
`rays[i] = new Ray(transform.position, Quaternion.Euler(new Vector3(0, ((angle_min - (angle_increment * i)) * 180 / Mathf.PI), 0)) * transform.forward);`
Basically we are calculating the subtraction in radians properly and then converting to degrees

**Comment by MartinBischoff** [comment]
Resolved in https://github.com/siemens/ros-sharp/commit/19088b2c25478e662ad6d735ab13215af366f565 .
Thank you @erdenbatuhan .
